src/Containers/BrainstormForm/BrainstormForm.tsx

```diff
@@ -73,6 +73,7 @@ const BrainstormForm:React.FC<Props> = ({brainstormFormState, cancel }) => {
     : writeError();
 
     history.push('/gameboard/round-one');
+    history.push('/instructions');
```

Comment:
will be sure to edit out the /gameboard/round-one path and make those uniform across the routings